blocks/hero-banner/hero-banner.css

```diff
@@ -258,6 +268,63 @@
         max-width: 56.5rem;
     }
 
+    .hero-banner-wrapper:has(.hero-banner.image-right) {
```

Comment:
I think `has` doesn't work with Firefox.

Reply:
Yeah, that's in the past. Now FF supports it ^ ^
<img width="487" alt="image" src="https://github.com/hlxsites/sunstar-foundation/assets/57093988/e849bb75-17f5-40da-b4a9-43be28c0aa48">

https://caniuse.com/css-has